src/components/Message/MessageListItem.tsx

```diff
@@ -1,29 +1,81 @@
 interface MessageListItemProps {
   user: User | null;
+  conversation: Conversation;
+  isSelected: boolean;
+  onClick: () => void;
 }
 
-export default function MessageListItem({ user }: MessageListItemProps) {
+export default function MessageListItem({
+  user,
+  conversation,
+  isSelected,
+  onClick,
+}: MessageListItemProps) {
+  const otherUser =
+    conversation.sender._id === user?._id
+      ? conversation.receiver
+      : conversation.sender;
+
+  const formatDate = (dateString: string): string => {
+    const date = new Date(dateString);
+    return date.toLocaleString("ko-KR", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+      hour: "2-digit",
+      minute: "2-digit",
+      hour12: false,
+    });
+  };
+
+  const formatMessageDateTime = formatDate(conversation.createdAt);
   return (
     <>
-      <div className="p-3 hover:bg-secondary rounded-lg my-1 transition-colors group">
+      <div
+        className={`p-3 rounded-lg my-1 transition-colors group cursor-pointer ${
+          isSelected ? "bg-primary" : "hover:bg-secondary"
+        }`}
+        onClick={onClick}
+      >
         <div className="flex items-center justify-between ">
           <div className="flex items-center justify-center">
             <img
-              src={user?.image || "/logo.png"}
+              src={otherUser.image || "/logo.png"}
               alt="프로필 이미지"
-              className="w-[30px] h-[30px] mr-3 mb-1 rounded-full profile"
+              className={`w-[30px] h-[30px] mr-3 mb-1 rounded-full profile transition-border ${
+                isSelected
+                  ? "border-gray-c8 bg-white/80"
+                  : "group-hover:border-gray-c8 group-hover:bg-white/80"
+              }`}
             />
-            <p className="text-lg dark:group-hover:text-gray-22 font-medium dark:text-white">
-              닉네임{user?.fullName}
+            <p
+              className={`text-lg font-medium ${
+                isSelected
+                  ? " dark:text-gray-22"
+                  : "dark:text-white dark:group-hover:text-gray-22"
+              }`}
+            >
+              {otherUser.fullName}
             </p>
           </div>
-          <p className="text-sm text-gray-6c  dark:text-gray-c8">
-            24.12.25 12:45
+          <p
+            className={`text-sm ${
+              isSelected
+                ? "text-gray-6c dark:text-gray-c6"
+                : "text-gray-6c dark:text-gray-c8 dark:group-hover:text-gray-c6"
+            }`}
+          >
+            {formatMessageDateTime}
```

Comment:
```suggestion
          <p
            className={`text-sm text-gray-6c dark:text-gray-c8 dark:group-hover:text-gray-6c ${
              isSelected && " dark:text-gray-6c"
            }`}
          >
            {formatMessageDateTime}
```

c6이라는 색상값이 없어서 제대로 적용되지 않은 것 같습니다!
그리고 이렇게 더 깔끔하게 만들어볼 수 있을 것 같아요